src/build/tenscript/bootstrap.scm

```diff
@@ -0,0 +1,73 @@
+;;;Seed
```

Comment:
A bit weird to put the names in comments but you know this. We'll get to including the in the (fabric ..) later

Reply:
Yes when you redo the parser.

Reply:
;)